buildrpm/go.mod.patch

```diff
@@ -0,0 +1,13 @@
+diff --git go.mod go.mod
+index 5d2e5659..68c81ebe 100644
+--- go.mod
++++ go.mod
+@@ -52,7 +52,7 @@ require (
+ 	go.opentelemetry.io/otel/trace v1.33.0 // indirect
+ 	go.uber.org/multierr v1.11.0 // indirect
+ 	go.uber.org/zap v1.27.0 // indirect
+-	golang.org/x/net v0.32.0 // indirect
```

Comment:
This needs the patch
replace golang.org/x/net => golang.org/x/net v0.33.0

Reply:
That occurs with this patch - buildrpm/go.mod.patch

Reply:
That is happening on line 10